Orso.Arpa.Application/ProjectApplication/ProjectDto.cs

```diff
@@ -19,6 +19,8 @@ public class ProjectDtoMappingProfile : Profile
         public ProjectDtoMappingProfile()
         {
             CreateMap<Project, ProjectDto>()
+                .ForMember(dest => dest.Title, opt => opt.MapFrom(src => src.Title))
```

Comment:
Da die Properties bei Description und Title in src und dest gleich heißen, kannst du die Zeilen 22 und 23 weglasssen

Reply:
erledigt